domain/repo.py

```diff
@@ -7,5 +7,7 @@
 class Repo(Protocol):
     """ Repo provides data access to Kentik Synthetic Tests """
 
-    def get_mesh_test_results(self, test_id: TestID, results_lookback_minutes: int) -> MeshResults:
+    def get_mesh_test_results(
+        self, test_id: TestID, results_lookback_minutes: int, timeout: Optional[Tuple[float, float]] = None
```

Comment:
Default timeout would be nice

Reply:
Maybe timeout should be configured in Repo initialization? I think it wont change from call to call

Reply:
Agree, moved setting timeout to init func